src/util/matcher/projector.js

```diff
@@ -0,0 +1,86 @@
+function projector(utils, e, array, obj) { // jshint ignore:line
```

Comment:
Hi @rexagod could this be abstracted to accept and produce coordinates independently from the Dom? It might be a good thing to add abstractly to the matcher-core lib?

Reply:
Okay, so just to clarify, you are suggesting that I should include this into the core lib, right? We can definitely do that and expose this into a parameter!

Reply:
I think that makes sense, as the desire to try to reconcile the two images into the same coordinate system is going to be a pretty common use case!